### PR TITLE
Verify warning log message when a build time property is updated

### PR DESCRIPTION
**Pull request by pjgg**
### Summary

Related to this upstream commit: https://github.com/quarkusio/quarkus/pull/25512

Please select the relevant options.
- [X] New scenario (non-breaking change which adds functionality)

### Checklist:
- [X] Methods and classes used in PR scenarios are meaningful
- [X] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java [hunk 1]
```diff
@@ -69,4 +69,21 @@ public void shouldDetectChanges() {
...
+        String newAppName = "new-test-app";
+        app.given()
+                .param("action", "updateProperty")
+                .param("name", "quarkus.application.name")
+                .param("value", newAppName)
+                .post("/q/dev/io.quarkus.quarkus-vertx-http/config")
+                .then().statusCode(200);
+
+        String expectedOutput = String.format("quarkus.application.name is set to '%s' but it is build time fixed to '%s'",
+                appName, newAppName);
```
Comment (jsmrcka) [inline http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java:86]: Is it just me, or should the message be formulated the other way around?
I mean, the test passes, but the message seems wrong to me. It should be e.g.
```
quarkus.application.name is set to 'new-test-app' but it is build time fixed to 'test-http'
```
There was a discussion about it in the GH issue (https://github.com/quarkusio/quarkus/issues/23573#issuecomment-1088517560) making the exact same point, which the second PR (https://github.com/quarkusio/quarkus/pull/25512) was supposed to fix.

Comment (pjgg) [reply http/http-minimum/src/test/java/io/quarkus/ts/http/minimum/DevModeHttpMinimumIT.java]: Updated!, thanks!

**Comment by pjgg** [comment]
Keep on hold this PR until [this issue](https://github.com/quarkusio/quarkus/issues/26156) get clarified

**Comment by pjgg** [comment]
This PR is not going to be merged, seems an issue in upstream that is already open

**Pull request closed without merging**